tests.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import quilter
+
+
+def test(method, csv_list):
+    output_root = r'c:\gis\elevation\statewide\test'
+    for csv_path in csv_list:
+        
+        csv_name = os.path.basename(csv_path).split('.')[0]
+        
+        if method == 'download':
+            test_dir = os.path.join(output_root, 'dl', csv_name)
+            args = [csv_path, test_dir]
+        
+        elif method == 'merge':
+            test_dir = os.path.join(output_root, 'merge', csv_name)
+            test_name = csv_name.split('.')[0]
+            args = [csv_path, test_dir, '-m', test_name]
+        
+        elif method == 'reproject':
+            test_dir = os.path.join(output_root, 'reproject', csv_name)
+            test_name = csv_name.split('.')[0]
+            args = [csv_path, test_dir, '-m', test_name, '-p', 'EPSG:3566']
+        
+        elif method == 'reproject_only':
+            test_dir = os.path.join(output_root, 'reproject_only', csv_name)
+            args = [csv_path, test_dir, '-p', 'EPSG:3566']
+
+        else:
+            raise NotImplementedError
+
+        quilter.main(args)
+
+
+def main():
+    csv_dir = r'c:\gis\git\quilter\csvs'
```

Comment:
this is going to need to be made relative so it works on my file system

Reply:
Fixed in next commit.